src/OpenSage.Game/Game.cs

```diff
@@ -423,11 +435,56 @@ public void StartGame(string mapFileName, IConnection connection, PlayerSetting[
             CumulativeLogicUpdateError = TimeSpan.Zero;
         }
 
+        public void StartCampaign(string side)
+        {
+            // TODO: Difficulty
+
+            var campaign = ContentManager.IniDataContext.Campaigns.Single(x => x.Name == side);
+            var firstMission = campaign.Missions.Single(x => x.Name == campaign.FirstMission);
+
+            StartGame(
+                firstMission.Map,
+                new EchoConnection(),
+                null,
+                0,
+                false);
+        }
+
+        public void StartMultiPlayerGame(
+            string mapFileName,
+            IConnection connection,
+            PlayerSetting[] playerSettings,
+            int localPlayerIndex)
+        {
```

Comment:
This will change in the future?

Reply:
It will need a lot more code than what's there now, if that's what you mean?